pyscf/scf/hf.py

```diff
@@ -80,6 +80,8 @@ def kernel(mf, conv_tol=1e-10, conv_tol_grad=None,
             generated by the builtin function :func:`locals`, so that the
             callback function can access all local variables in the current
             environment.
+        sapgrasp : str (small | large)
+            SAP initial guess coefficient set size.
```

Comment:
This is a bad name for the parameter controlling the potential basis, since many parametrizations have been suggested in the literature, and it can also be customized by the user.